app/views/shared/_header.html.erb

```diff
@@ -56,10 +53,10 @@
       <% if signed_in? %>
         <li class="nav-item mr-auto">
           <% if current_admin_user.present? %>
-            <%= render partial: 'shared/logout', locals: { _logout_path: destroy_admin_user_session_path, _account_badge_letter: 'A' } %>
+            <%= render partial: 'shared/logout', locals: { _logout_path: destroy_admin_user_session_path } %>
           <% end %>
           <% if current_user.present? %>
-            <%= render partial: 'shared/logout', locals: { _logout_path: destroy_user_session_path, _account_badge_letter: 'U' } %>
+            <%= render partial: 'shared/logout', locals: { _logout_path: destroy_user_session_path } %>
```

Comment:
why is it an `_` path?

Reply:
it's not new tho, so nvm

Reply:
Prefixing with `_` is a convention sometimes used to denote local variables in a partial